dbms/src/Common/CurrentThread.cpp

```diff
@@ -26,6 +26,11 @@ void CurrentThread::updatePerformanceCounters()
     current_thread->updatePerformanceCounters();
 }
 
+bool CurrentThread::isInitialized()
+{
+    return likely(current_thread);
```

Comment:
likely make no sense here.
It only make sense for branches (that at least require inlining).